config/browser-extension/ext.prod.config.js

```diff
@@ -2,15 +2,12 @@ const merge = require('webpack-merge');
 const CreateFileWebpack = require('create-file-webpack');
 
 const common = require('./ext.base.config');
+const packageConf = require('../../package');
 
 // Constant key and id from our production setting
 const KEY = 'MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEArkmF/06jK2b976qLc1RoSyrhGMN1QkbRCbDLnMGL+RfOlZdbDBwXxtTXcnzm2uwUlrZKCaecuD0GOrnOfihaqQv3cVRbRTCWhvyhswqiWea8YA8P0pjiTrsuth6l0NK9WqKYLTUxP+oM9H0nDaOg5I9Jzh5kCZPpwavbx1naXCv7PPcIoGNRgA4y3YJL8qiJx9eY0I6yRdCSS8JX4AeZ4K1ShnzPg5Wih7NZdJlmWCjOajdF8GHxYGt68eGOamIlQYwFzAjinCN+DbI1f4suQcq5oJzw3NCBmCvepaw5f/ivhzCsw0aRfVqRk6EVxhQvRwTBxeBQQ28j5JecxtfN5wIDAQAB'
```

Comment:
I suppose we could also move the key `packageConf` -- they are very closely related and whenever one changes, so does the other (not that we expect it to happen any time soon, but...)